app/(tabs)/index.tsx

```diff
@@ -1,78 +1,135 @@
 import { Text, View, } from '../../components/Themed';
-import { TextInput } from 'react-native-gesture-handler';
-import { Button } from 'react-native';
 import { Component } from 'react';
+import { FlatList } from 'react-native-gesture-handler';
+import { FontAwesome5 } from '@expo/vector-icons';
+import Swipeable from 'react-native-gesture-handler/Swipeable';
 
-import useGun from '../../hooks/useGun';
-const { gun, app, user, SEA } = useGun();
+const DATA = [
+  {
+      name: 'Bananas',
+      data: [{user: "Test bruger", date: "2020.11.01", price: 3}],
+  },
+  {
+    name: 'Chorizo',
+    data: [{user: "Andreas", date: "2020.11.01", price: 35}],
+  },
+  {
+    name: 'Beans',
+    data: [{user: "Mike", date: "2020.11.01", price: 2}],
+  },
+  {
+    name: 'Apple',
+    data: [{user: "Emil", date: "2020.11.01", price: 2.5}],
+  },
+  {
+    name: 'Chocolate',
+    data: [{user: "Martin", date: "2020.11.01", price: 13.95}],
+  },
+]
 
-type Props = {
-  text: any
+type ListData = { 
+  name: string, 
+  data: { user: string, date: string, price: number }[]
 }
 
-type State = {
-  text: string
-  name: string,
-  paste: string
+function renderItem({item, index}: { item: ListData, index: number}) {
+  return (
+    <Swipeable
+      friction={3}
+      renderLeftActions={leftSwipeAction}
+      renderRightActions={rightSwipeAction}
+      onSwipeableOpen={swipeHandler}>
+      <View style={[styles.container, 
+        {backgroundColor: index % 2 == 0 ? '#eeeeee' : '#D3D3D3'}]}>
+        <View style={styles.item}>
+          <Text style={styles.itemText}>{item.name}</Text>
+          <Text style={styles.itemPrice}>{item.data[0].price} kr.</Text>        
+        </View>
+        <Text style={styles.infoText}>Added by {item.data[0].user} {item.data[0].date}</Text>
+      </View>
+    </Swipeable>
+  )
 }
 
-export default class ListScreen extends Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      text: 'Whats your name?',
-      name: '',
-      paste: '',
-    };
-  }
+function leftSwipeAction() {
+  return (
+    <View style={{backgroundColor: '#5CBCA9', justifyContent: 'center', alignItems: 'flex-end'}}>
+      <FontAwesome5
+        name="arrow-alt-circle-right" 
+        size={32} 
+        color="white"
+        style={styles.swipe}
+        />
+    </View>
+  )
+}
 
-  $user = user.create('test', 'testtest', () => {})
+function rightSwipeAction() {
+  return (
+    <View style={{backgroundColor: '#E35F52', justifyContent: 'center', alignItems: 'flex-end'}}>
+      <FontAwesome5
+        name="trash-alt" 
+        size={32} 
+        color="white"
+        style={styles.swipe}
+        />
+    </View>
+  )
+}
 
-  componentDidMount(): void {
-    this.$user;
-    app.on((data: any) => {
-      console.log('data', data);
-      this.setState({paste: data.paste});
-    });
+function swipeHandler(dir: 'left' | 'right') {
+  if (dir == "left") {
+      alert(dir);
```

Comment:
The constant alerting is kinda annoying maybe just log to console until proper implementation is done